# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/xf86-video-sunffb/hotfix-ramdac.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- xf86-video-sunffb-1.2.2/src/ffb.h.vanilla	2022-08-13 16:41:01.922816508 +0200
+++ xf86-video-sunffb-1.2.2/src/ffb.h	2022-08-13 16:44:21.816808537 +0200
@@ -30,7 +30,7 @@
 
 #include "xf86.h"
 #include "xf86_OSproc.h"
-#include "xf86RamDac.h"
+#include "xf86Cursor.h"
 #ifdef HAVE_XAA_H
 #include "xaa.h"
 #endif
